Doc/library/functions.rst

```diff
@@ -1487,7 +1487,7 @@ are always available.  They are listed here in alphabetical order.
    unless the second argument is negative; in that case, all arguments are
    converted to float and a float result is delivered.  For example, ``pow(10, 2)``
    returns ``100``, but ``pow(10, -2)`` returns ``0.01``.  For a negative base of
-   type :class:`int` or :class:`float` and a non-integral exponent, a complex
+   type :class:`int` or :class:`float` and a non-integer exponent, a complex
```

Comment:
I'll wait to get a review from @mdickinson to know what he thinks before making a dedicated PR for this change.

Reply:
So I agree that `non-integer` is more likely to be immediately meaningful to readers than `non-integral`.

The problem I have with it in this context is that to me (and I suspect I'm not the only one) it's too suggestive of type rather than value. That is, if I'm not thinking too hard then I'd think that of course `2.0` isn't an integer (it's a float!), so the conclusion of the sentence would apply to an exponent of `2.0`. But that's wrong: we want to convey the idea that the _value_ is not an integer, regardless of the type.

Can you suggest an alternative way to word this that makes it clear that "non-integer" applies to `2.3` but not to `2.0`?

Reply:
Maybe:
```
... and an exponent whose value has a nonzero fractional part
```
?

Reply:
Thanks for chiming in, Mark! With that in mind, I (FWIW) prefer the existing wording. We can instead consider to expand the example in L1491.